.github/workflows/release.yml

```diff
@@ -0,0 +1,26 @@
+name: Release
+
+on:
+  push:
+    # Run on pushes only to master
+    branches:
+      - master
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Setup Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - name: Install dependencies
+        run: yarn install
```

Comment:
Not sure it will work. Maybe we should just switch to npm? I remember one of the reasons for using yarn was tests not running on windows but I can't remember exactly anymore.

Reply:
I don't think we need to install deps, since the semantic-release action should include its own deps. I think the only missing thing was the checkout action.

Reply:
I think we do since we have our own dependencies like Listr